packages/istio/charts/istio-1.6.3/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: istio-1.6.3
+version: 0.1.0
```

Comment:
we may want to follow the pattern of version the chart at the app version with "00" appended to the end. I think dev charts uses this version. cc @cbron